package.json

```diff
@@ -14,6 +14,8 @@
     "@types/react-mailchimp-subscribe": "^2.1.1",
     "@types/react-router-dom": "^5.3.2",
     "axios": "^0.26.1",
+    "bignumber.js": "^9.0.2",
+    "dayjs": "^1.11.1",
```

Comment:
Just wonder if we really need `dayjs` lib. I was able to format the date in `ProposalCard` using `Date` object. Please see https://github.com/threshold-network/splash-page/blob/threshold-site-2.0/src/templates/home-page/ActiveCommunity/ProposalCard.tsx#L23-L26 Ofc up to you I don't wanna be a blocker 😆

Reply:
yeah good point. I've created a `useFormatDate` hook to share the logic